packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -158,13 +157,21 @@ class LinearProgressIndicator extends ProgressIndicator {
   /// The [value] argument can be either null (corresponding to an indeterminate
   /// progress indicator) or non-null (corresponding to a determinate progress
   /// indicator). See [value] for details.
+  ///
+  /// If [height] is less than 1.0 then a value of 1.0 will be used.
```

Comment:
why?

Reply:
Which fallback value do you prefer?

Is `< 1` pixel logical?

Reply:
1.0 pixels is about three actual physical pixels on some devices. Also if you want to animate the height from nothing, you'll need to start with small numbers like 0.01, 0.1, 0.5, etc.